scripts/Imaging/imgpy/filters/stripe_removal.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import (absolute_import, division, print_function)
+# Remove stripes in sinograms / ring artefacts in reconstructed volume
+import helper as h
+
+
+def cli_register(parser):
+    parser.add_argument(
+        "--pre-stripe-removal-wf",
+        nargs='*',
+        required=False,
+        type=str,
+        help="Stripe removal using wavelett-fourier method. Available parameters:\n\
+        level (int, optional) Number of discrete wavelet transform levels.\n\
+        wname (str, optional) Type of the wavelet filter. 'haar', 'db5', 'sym5', etc.\n\
+        sigma (float, optional) Damping parameter in Fourier space.\n\
+        pad (bool, optional) If True, extend the size of the sinogram by padding with zeros."
+    )
+
+    parser.add_argument(
+        "--pre-stripe-removal-ti",
+        nargs='*',
+        required=False,
+        type=str,
+        help="Stripe removal using Titarenko's approach. Available parameters:\n\
+              nblock (int, optional) Number of blocks.\n\
+              alpha (int, optional) Damping factor.")
+
+    parser.add_argument(
+        "--pre-stripe-removal-sf",
+        nargs='*',
+        required=False,
+        type=str,
+        help="Stripe removal using smoothing-filter method. Available parameters:\n\
+        size (int, optional) Size of the smoothing filter.")
+
+    return parser
+
+
+def gui_register(par):
+    raise NotImplementedError("GUI doesn't exist yet")
+
+
+def methods():
+    return [
+        'wf', 'wavelet-fourier', 'ti', 'titarenko', 'sf', 'smoothing-filter'
+    ]
+
+
+def execute(data, wf, ti, sf, cores=None, chunksize=None):
+    # get the first one, the rest will be processed
+    msg = "Starting removal of stripes/ring artifacts using the method '{0}'..."
+    if wf is not None:
```

Comment:
`if wf`
`elif ti`
`elif sf`

Also do we need an `else` in case none were true?